msprime/cli.py

```diff
@@ -121,15 +161,21 @@ def __init__(
             population_configurations=None, demographic_events=None,
             scaled_mutation_rate=0, print_trees=False,
             precision=3, random_seeds=None,
-            scaled_gene_conversion_rate=0, gene_conversion_track_length=1):
+            scaled_gene_conversion_rate=0, gene_conversion_track_length=1,
+            hotspots=None):
         self._sample_size = sample_size
         self._num_loci = num_loci
         self._num_replicates = num_replicates
         self._recombination_rate = scaled_recombination_rate
         self._mutation_rate = scaled_mutation_rate
         # For strict ms-compability we want to have m non-recombining loci
-        recomb_map = msprime.RecombinationMap.uniform_map(
-            num_loci, self._recombination_rate, num_loci)
+        if hotspots is None:
+            self._recomb_map = msprime.RecombinationMap.uniform_map(
+                num_loci, self._recombination_rate, num_loci)
+        else:
+            self._recomb_map = hotspots_to_recomb_map(
+                    hotspots, scaled_recombination_rate, num_loci)
```

Comment:
probably better to use ``self._recombination_rate`` so the reader isn't wondering why the two cases are different.